resource/layout/steamrootdialog.layout

```diff
@@ -57,7 +57,7 @@
 		{
 			ControlName=MenuButton
 			style="inbox_button"
-			textAlignment=west
+			textAlignment=east

```

Comment:
I though this would right-align the notification counter, but I think the counter "bounding box" must be extended too, right?

Somewhere on line 446